apps/web/src/pages/Profile/ResumeAndRecruitmentsPage/components/ResumeAndRecruitments.tsx

```diff
@@ -0,0 +1,229 @@
+        },
+      )}
+      title={intl.formatMessage(titles.resumeAndRecruitments)}
+      leaveRoomForNavigation
+    >
+      <TableOfContents.Wrapper>
+        <TableOfContents.Navigation>
+          <TableOfContents.AnchorLink id={PAGE_SECTION_ID.MANAGE_YOUR_RESUME}>
+            {intl.formatMessage(titles.manageYourResume)}
+          </TableOfContents.AnchorLink>
+          <TableOfContents.AnchorLink
+            id={PAGE_SECTION_ID.QUALIFIED_RECRUITMENT_PROCESSES}
+          >
+            {intl.formatMessage(titles.qualifiedRecruitmentProcesses)}
+          </TableOfContents.AnchorLink>
+        </TableOfContents.Navigation>
+        <TableOfContents.Content>
+          <TableOfContents.Section id={PAGE_SECTION_ID.MANAGE_YOUR_RESUME}>
+            <Heading Icon={BookmarkSquareIcon} color="tertiary">
+              {intl.formatMessage(titles.manageYourResume)}
+            </Heading>
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "This section is similar to your traditional résumé and describes your experiences across work, school, and life. You’ll be able to reuse this information on each application you submit on the platform, speeding up the process and ensuring that your information is always up-to-date.",
+                id: "gPy2MA",
+                description:
+                  "Descriptive paragraph for the Manage your resume section of the résumé and recruitments page.",
+              })}
+            </p>
+            {missingSkills && (
+              <div data-h2-margin="base(x1, 0)">
+                <MissingSkills
+                  addedSkills={
+                    hasResumeItems ? flattenExperienceSkills(experiences) : []
+                  }
+                  requiredSkills={missingSkills.requiredSkills}
+                  optionalSkills={missingSkills.optionalSkills}
+                />
+              </div>
+            )}
+            <div data-h2-margin-top="base(x2)">
+              <ResumeSection
+                editParam={applicationParam}
+                experiences={experiences}
+                headingLevel="h2"
+                applicantId={applicantId}
+              />
+            </div>
+          </TableOfContents.Section>
+          <TableOfContents.Section
+            id={PAGE_SECTION_ID.QUALIFIED_RECRUITMENT_PROCESSES}
+            data-h2-margin="base(x3, 0, 0, 0)"
+          >
+            <div data-h2-flex-grid="base(center, x1, x1)">
+              <Heading
+                Icon={IdentificationIcon}
+                color="secondary"
+                data-h2-flex-item="base(1of1) p-tablet(fill)"
+              >
+                {intl.formatMessage(titles.qualifiedRecruitmentProcesses)}
+              </Heading>
+              <Link
+                href={paths.browsePools()}
+                data-h2-flex-item="base(1of1) p-tablet(content)"
+              >
+                Browse jobs
+              </Link>
```

Comment:
Two things for this one:

1. mode should be set to `inline`
2. Need the "Browse jobs" to be wrapped in `intl.formatMessage`